receiver/oktareceiver/go.mod

```diff
@@ -0,0 +1,73 @@
+module github.com/observiq/bindplane-agent/receiver/oktareceiver
+
+go 1.23.0
```

Comment:
Let's use go 1.21.9 or go 1.22.6 (everything is using 1.21.9 right now, but we'll be updating to 1.22 soon)